openbas-front/src/admin/components/simulations/simulation/injects/ExerciseInjects.tsx

```diff
@@ -165,7 +165,7 @@ const ExerciseInjects: FunctionComponent<Props> = () => {
         switch (action.type) {
           case 'ADD':
             if (isNotEmptyField(injectToUpdate[`inject_${action.field}`])) {
-              injectToUpdate[`inject_${action.field}`] = R.uniq([...injectToUpdate[`inject_${action.field}`], action.values.map((n) => n.value)]);
+              injectToUpdate[`inject_${action.field}`] = R.uniq([...injectToUpdate[`inject_${action.field}`], ...action.values.map((n) => n.value)]);
             } else {
```

Comment:
I also made this fix in scenario injects on the bulk delete PR.

Reply:
Great, the conflict will be easy to manage :)